components/brave_wallet/browser/json_rpc_service_base.h

```diff
@@ -0,0 +1,43 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_WALLET_BROWSER_JSON_RPC_SERVICE_BASE_H_
+#define BRAVE_COMPONENTS_BRAVE_WALLET_BROWSER_JSON_RPC_SERVICE_BASE_H_
+
+#include <string>
+#include <vector>
+
+#include "brave/components/api_request_helper/api_request_helper.h"
+#include "brave/components/brave_wallet/common/brave_wallet.mojom.h"
+
+namespace brave_wallet {
+class EnsResolverTask;
+
+class JsonRpcServiceBase {
```

Comment:
Could you explain what was the reason adding this base class?

Reply:
removed base class